src/Query/QueryLockBuilder.php

```diff
@@ -0,0 +1,75 @@
+<?php
+
+declare(strict_types=1);
+
+namespace Doctrine\DBAL\Query;
+
+use Doctrine\DBAL\Platforms\AbstractPlatform;
+use Doctrine\DBAL\Platforms\SQLServerPlatform;
+
+use function implode;
+use function ksort;
+use function ltrim;
+use function rtrim;
+use function trim;
+
+/** @internal */
+final class QueryLockBuilder
```

Comment:
Would it make sense to merge this class with `QueryBuilder`? On its own, this class does not make much sense, does it?

Reply:
Removing the `$this->platform instanceof SQLServerPlatform`, as you requested, made this class disappear.